<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="wMain">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="wLeftArea" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splSecondary">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTabWidget" name="twDocuments">
           <property name="tabPosition">
            <enum>QTabWidget::North</enum>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="usesScrollButtons">
            <bool>true</bool>
           </property>
           <property name="documentMode">
            <bool>true</bool>
           </property>
           <property name="tabsClosable">
            <bool>true</bool>
           </property>
           <property name="movable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="WCodeEditor" name="plainTextEdit"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
          <widget class="QWidget" name="wConsoleArea" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wRightArea" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mbMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScript">
    <property name="title">
     <string>S&amp;cript</string>
    </property>
    <widget class="QMenu" name="menuScriptRecent">
     <property name="title">
      <string>&amp;Recent scripts</string>
     </property>
     <property name="icon">
      <iconset theme="document-open-recent">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <addaction name="actionScriptRecent_None"/>
    </widget>
    <addaction name="actionScriptNew"/>
    <addaction name="actionScriptOpen"/>
    <addaction name="menuScriptRecent"/>
    <addaction name="separator"/>
    <addaction name="actionScriptSave"/>
    <addaction name="actionScriptSaveAs"/>
    <addaction name="actionScriptSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionScriptClose"/>
    <addaction name="separator"/>
    <addaction name="actionScriptExecute"/>
    <addaction name="separator"/>
    <addaction name="actionScriptQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>E&amp;dit</string>
    </property>
    <addaction name="actionEditUndo"/>
    <addaction name="actionEditRedo"/>
    <addaction name="separator"/>
    <addaction name="actionEditCut"/>
    <addaction name="actionEditCopy"/>
    <addaction name="actionEditPaste"/>
    <addaction name="separator"/>
    <addaction name="actionEditSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Lan&amp;guage</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Setti&amp;ngs</string>
    </property>
    <addaction name="actionSettingsPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionSettingsSaveSessionOnExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionHelpBuliScriptHandbook"/>
    <addaction name="separator"/>
    <addaction name="actionHelpAboutBS"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewShowCanvas"/>
    <addaction name="actionViewShowCanvasOrigin"/>
    <addaction name="actionViewShowCanvasGrid"/>
    <addaction name="actionViewShowCanvasPosition"/>
    <addaction name="separator"/>
    <addaction name="actionViewShowConsole"/>
   </widget>
   <addaction name="menuScript"/>
   <addaction name="menuEdit"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="sbInfo"/>
  <action name="actionHelpBuliScriptHandbook">
   <property name="icon">
    <iconset theme="help-contents">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;BuliScript handbook...</string>
   </property>
  </action>
  <action name="actionHelpAboutBS">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About Buli Script...</string>
   </property>
  </action>
  <action name="actionScriptNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="actionScriptOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionScriptSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionScriptSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;ve as...</string>
   </property>
  </action>
  <action name="actionScriptSaveAll">
   <property name="icon">
    <iconset theme="document-save-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save a&amp;ll</string>
   </property>
  </action>
  <action name="actionScriptExecute">
   <property name="icon">
    <iconset theme="run-build">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Execute</string>
   </property>
  </action>
  <action name="actionScriptQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionScriptClose">
   <property name="icon">
    <iconset theme="document-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionScriptRecent_None">
   <property name="text">
    <string>(&amp;no recent scripts)</string>
   </property>
  </action>
  <action name="actionEditCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="actionEditCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
  </action>
  <action name="actionEditPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionEditSelectAll">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Select all</string>
   </property>
  </action>
  <action name="actionEditUndo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionEditRedo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionSettingsPreferences">
   <property name="icon">
    <iconset theme="settings-configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionViewShowCanvasGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show canvas &amp;grid</string>
   </property>
  </action>
  <action name="actionViewShowCanvasOrigin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;canvas origin</string>
   </property>
  </action>
  <action name="actionViewShowCanvasPosition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show canvas &amp;position</string>
   </property>
  </action>
  <action name="actionViewShowCanvas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show canvas</string>
   </property>
  </action>
  <action name="actionViewShowConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;how console</string>
   </property>
  </action>
  <action name="actionSettingsSaveSessionOnExit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Save session on exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WCodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>buliscript.pktk.widgets.wcodeeditor</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
